src/generate_letters.ml

```diff
@@ -0,0 +1,33 @@
+(** letter_gen is a list containing 7 random chars at any time.
+    To be used in a game of scrabble. If the char has been used in the game,
+    there will be an '\000' in its place.
+    Requires each char in the list to be an alphabetical letter or '\000'. *)
```

Comment:
Instead of replacing chars with the null char, consider making a method that takes the char at index `n`, replaces it with a new random char, and returns the original one. This way, it is impossible to end up with an invalid `letter_gen` structure. Also, since the `letter_gen` structure does not actually generate anything, a better name might be `letter_buf` or `letter_deck`.